jobs/config.json

```diff
@@ -892,6 +892,16 @@
   ]
 },
 
+"ci-kubernetes-e2e-prow-canary": {
+  "scenario": "kubernetes_prow",
```

Comment:
I suspect it will be easier to simply pass a --k8s flag or whatever to the existing kubernetes_e2e scenario.